third_task.py

```diff
@@ -0,0 +1,7 @@
+#3 task
+casino_blacklist = ['Franklin Gates', 'Jeremy Gallagher', 'Tahlia Cuevas', 'Keane Lane' ]
+poker_blacklist = ['Genevieve Owens', 'Ellena Petty', 'Franklin Gates', 'Shawn Gutierrez', 'Asad Villega', 'Stella Mays']
+alcohol_blacklist = ['Kylie Valdez', 'Lawrence Calhoun', 'Zaina Holland', 'Franklin Gates', 'Anna Payne', 'Ruby Hatfield']
+print(casino_blacklist.count('Franklin Gates'))
+print(poker_blacklist.count('Franklin Gates'))
+print(alcohol_blacklist.count('Franklin Gates'))
```

Comment:
Why did you check only Franklin Gates?
you just hard-coded data. and the data type of this blacklist is not correct. Usually, a black list is a list of unique data. 
Please find the correct type and use it functions for the solution.